src/lib/Util/adapters/bbc-kaldi/groups-words-by-speakers.test.js

```diff
@@ -0,0 +1,25 @@
+import groupWordsInParagraphsBySpeakers from './group-words-by-speakers';
+
+import kaldiTedTalkTranscript from './sample/bbcKaldiTranscriptWithSpeakerSegments.sample.json';
+
+const segmentation = kaldiTedTalkTranscript.retval.segmentation;
+const words = kaldiTedTalkTranscript.retval.words;
+
+describe('groupWordsInParagraphsBySpeakers', () => {
+  /**
+     * Hard to test if the segmentation algo it's working properly
+     * But one basic test for now is to test there is the same number of words
+     * In the result.
+     */
+  it('Expect same word count in results', ( ) => {
+
+    const wordsByParagraphs = groupWordsInParagraphsBySpeakers(words, segmentation);
+
+    const resultWordCount = wordsByParagraphs.reduce(reduceFunction, 0);
+    function reduceFunction(total, currentParagraph) {
```

Comment:
cool, could be a const function but no big deal if not

Reply:
ok, will leave for now